packages/trpc/server/document-router/router.ts

```diff
@@ -281,7 +282,12 @@ export const documentRouter = router({
           query,
           userId: ctx.user.id,
         });
-        return documents;
+
+        const maskedDocuments = documents.map((document) =>
+          maskRecipientTokensForDocument({ document, user: ctx.user }),
+        );
```

Comment:
What's this for? We're already masking the token inside the function

The owner of the document gets to see all the tokens